index.js

```diff
@@ -70,6 +70,38 @@ Builder.prototype.build = function(moduleName, outFile, opts) {
   });
 };
 
+var operatorFunctions = {
+  '+': builder.addTrees,
+  '-': builder.subtractTrees
+};
+
+Builder.buildExpression = function(firstModule, operations, cfg) {
```

Comment:
Doesn't handle rich expressions at all, need some recursion for that. Not sure the value of grouping with just `+` and `-`, unless I'm missing something you can achieve the same effects with reordering and maybe repeating in perverse cases?

New operators coming?